src/types/formatted/healthCareLocalFacility.ts

```diff
@@ -10,7 +10,7 @@ import { PublishedEntity } from './publishedEntity'
 import { FormattedRelatedLinks } from './relatedLinks'
 import { SideNavMenu } from './sideNav'
 import { MediaImage } from '@/types/formatted/media'
-import { Wysiwyg } from './wysiwyg'
+import { FacilitySocialLinksProps } from '@/templates/layouts/healthCareLocalFacility/FacilitySocialLinks'
```

Comment:
I don't love pulling this type in from the template, but it _does_ make it explicit that the data is _meant_ to be fed into the component. 🤷 If you've got strong opinions, let me know and I can refactor.